src/MuxManager.cpp

```diff
@@ -575,4 +575,21 @@ void MuxManager::handleWarmRestartReconciliationTimeout(const boost::system::err
     mDbInterfacePtr->setWarmStartStateReconciled();
 }
 
+//
+// ---> handleTsaEnableNotification
+//
+// handle TSA Enable Notification
+//
+void MuxManager::handleTsaEnableNotification(bool enable)
+{
+    if (enable) {
+        PortMapIterator portMapIterator = mPortMap.begin();
+
+        while (portMapIterator != mPortMap.end()) {
+            portMapIterator->second->handleTsaEnable();
+            portMapIterator ++;
+        }
+    }
```

Comment:
When TSB is issued, we should come back to active, right?

Reply:
Add support to come back to `auto`. This way we will let the state machine to decide if it needs to toggle to active. And the behavior aligns with explicitly executing TSB. 

https://github.com/Azure/sonic-buildimage/pull/9726